ui/app/[locale]/kafka/[kafkaId]/@header/topics/page.tsx

```diff
@@ -28,8 +29,12 @@ async function ConnectedHeader({ params }: { params: KafkaParams }) {
   return (
     <Header
       total={topics.meta.page.total}
-      ok={topics.meta.page.total}
-      warning={0}
+      ok={topics.meta.summary.statuses.FullyReplicated ?? 0}
+      warning={topics.meta.summary.statuses.UnderReplicated ?? 0}
+      error={
+          (topics.meta.summary.statuses.PartiallyOffline ?? 0) +
+          (topics.meta.summary.statuses.Offline ?? 0)
+      }
```

Comment:
```suggestion
      ok={topics.meta.summary.statuses.FullyReplicated || 0}
      warning={topics.meta.summary.statuses.UnderReplicated || 0}
      error={
          (topics.meta.summary.statuses.PartiallyOffline || 0) +
          (topics.meta.summary.statuses.Offline || 0)
      }
```

Just a preference, I don't see that style used often in other js/ts codebases

Reply:
Yeah, I don't have a preference. The `||` operator is flagged by my IDE and it says to prefer the nullish coalescing operator `??` "because it's safer". It seems to be something that is relatively newer.

Reply:
TIL https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing

I guess I'll start using it as well then, I have surely been bitten by a default swallowing a valid 0 before